vsg/token/range.py

```diff
@@ -1,20 +0,0 @@
-
-from vsg import parser
```

Comment:
Hey @Emilv2 ,

I would like to keep this file for when I eventually enhance my parser.  If you could undelete it, I will merge in the pull request.

Thanks,

--Jeremy